tools/simulator/metrics-collector/generate-metrics-data.sh

```diff
@@ -40,47 +41,63 @@ if ! command -v kubectl &>/dev/null; then
     if [[ "$(uname)" == "Linux" ]]; then
       curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.21.0/bin/linux/amd64/kubectl
     elif [[ "$(uname)" == "Darwin" ]]; then
-      curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.21.0/bin/darwin/amd64/kubectl
```

Comment:
I previously created and consolidated the tooling into https://github.com/stolostron/multicluster-observability-operator/blob/main/scripts/install-binaries.sh

Would it help to extend that script with the requirements and call as needed?

Reply:
Yes, that seems sensible, will take a look :+1:

Reply:
Ok done!